extensions/wikia/AffiliateService/js/ext.AffiliateService.js

```diff
@@ -340,7 +340,9 @@ require([
 				}
 			});
 
-			AffiliateService.insertAtPointAndTrack($insertionPoint, unit);
+			if ($insertionPoint !== undefined) {
```

Comment:
This already exists in other function because we're firing no-impression otherwise.
```
		insertAtPointAndTrack: function ($insertionPoint, unit) {
			// check if we found good insertion point
			if ($insertionPoint) {
```